class-components/src/utils/ui/Button/Button.tsx

```diff
@@ -0,0 +1,9 @@
+import { Component, ReactNode } from "react";
+
+export class Button extends Component {
+  render(): ReactNode {
+    return <>
+      <button type="submit">Search!</button>
+    </>
```

Comment:
```suggestion
    return <button type="submit">Search!</button>
```
фрагменты используются только если ты возвращаешь несколько элементов, например два дива, и не хочешь их ни во что оборачивать (в третий див). если ты возвращаешь только один элемент ;в нем может быть сколько угодно чего хочешь) то фрагмент не нужен

Reply:
Понял! Всё исправлю. Спасибо!